app/lib/person_gross_income_subtotals.rb

```diff
@@ -52,6 +52,7 @@ def monthly_unspecified_source
   end
 
   def is_student?
+    # Student check is done via presence of irregular_incomes where income_type == student_loan (Request payload: irregular_incomes > payments[] > {income_type: student_loan})
```

Comment:
I thought we already had a comment for this? The idea was that it looks wrong to check is_student? by looking at the payments, but the guidance says just that so the comment should reflect that